rust/lance-encoding/src/encodings/physical/value.rs

```diff
@@ -27,14 +30,38 @@ pub struct ValuePageScheduler {
     // I think we want to, in theory, but will need to test this case.
     bytes_per_value: u64,
     buffer_offset: u64,
+    buffer_size: u64,
+    compressed: bool,
 }
 
 impl ValuePageScheduler {
-    pub fn new(bytes_per_value: u64, buffer_offset: u64) -> Self {
+    pub fn new(bytes_per_value: u64, buffer_offset: u64, buffer_size: u64) -> Self {
+        let compressed = std::env::var("LANCE_COMPRESSED_PAGE").is_ok();
```

Comment:
We can use an environment variable on the write side.  However, we should not use the environment variable on the read side.  I think changes to `encodings.proto` are needed.  For now, we can probably do this:

```
// Fixed width items placed contiguously in a buffer
message Flat {
  // the number of bits per value, must be greater than 0, does
  // not need to be a multiple of 8
  uint64 bits_per_value = 1;
  // the buffer of values
  Buffer buffer = 2;
  // The Compression message can specify the compression scheme (e.g. zstd) and any
  // other information that is needed for decompression.
  Compression compression = 3;
}
```

Later, if we end up going with the "dedicated thread pool for compression" and make compression part of the I/O scheduler then I think the `Compression` message will be a part of `Buffer` and not part of `Flat`.  However, we can tackle that later.

Reply:
I added such a structure in protobuf:
```
message Compression {
  string scheme = 1;
}
```
Currently, only the name of the compression scheme is stored and retrieved. Let me know if this is okay. Thanks.